### ISSUE TITLE
Build list view for jobs/resources

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/alnap#59]

**Comment by nadia847** [comment]
ALNAP resources to come up first in results 
Sorting of results - @dave-manta is this just another way of saying they need to be able to sort results?

**Comment by dave-manta** [comment]
@nadia847 I'm not sure it is. They asked for ALNAP resources to be prioritised above others, however I'm not sure that works.  Another one to be asked on Slack.... (I'll ask now).

**Comment by dave-manta** [comment]
Awaiting client repsonse.

**Comment by dave-manta** [comment]
Order of default search results... ALNAP, then Member, then others marked with neither.

**Comment by nadia847** [comment]
@dave-manta don't think that'll work, because when all resources appear at first on the search page, all alnap ones will be first, by date, then after all of those, the members ones descending order  by date and then after that the rest of the resources. 

so we could have the search page display nothing but filters until they search, ie it's a separate page that you go to only after you have run a search 

if that's not good enough, it'll need custom development? @meldera @russ-manta ? can we do that?

**Comment by dave-manta** [comment]
@nadia847 not sure I follow, but please read the thread Q6 on client-alnap channel in slack to see if that helps.

**Comment by angievanhegan** [comment]
Q6
@mgili you have asked for ALNAP resources to come up in first in results.  Does that mean you want ALL ALNAP resources followed by external resources?


19 replies
mgili [4 days ago] 
dave: Yes, all relevant ALNAP resources to that particular search, of course

dave [4 days ago] 
And what if you search for All resources, have all those marked as ALNAP first? @mgili

mgili [4 days ago] 
Yep

dave 
[4 days ago] 
OK :slightly_smiling_face:

dave 
[4 days ago] 
And if a user sorted results say by date would you want to list all ALNAP by newest to oldes followed by members newest to oldest and then others newest to oldest, or would we abandon the ALNAP priority?

mgili [4 days ago] 
I think for results by date we would abandon the ALNAP priority. We can only tweak the search so much!

dave [4 days ago] 
I agree.

dave 
[4 days ago] 
Oh no, I have another question.  What is the default order for resource results... by date? http://www.alnap.org/resources/results.aspx?q=%7Cta

mgili [4 days ago] 
I'd say by relevance as default, with ALNAP resources coming first + have the option of results by date

mgili [4 days ago] 
available

dave 
[4 days ago] 
And how do you define relevance when viewing ALL resources?

mgili [4 days ago] 
What would you recommend?

dave 
[4 days ago] 
Good question...

mgili [4 days ago] 
We think it may make sense to go with results by date only in these scenarios? Just putting ourselves in the shoes of a user who would do that. The aim for a "no keyword" search would be to find out what's the latest that has been put up

mgili [4 days ago] 
what do you think?

dave 
[4 days ago] 
Was thinking of date or popularity.  But probably date.

mgili [4 days ago] 
I'd definitely go with date. The most popular resources in the HELP, at least according to Phil's stats, are quite random tbh

mgili [4 days ago] 
(which has always made us wonder about their accuracy)

dave 
[4 days ago] 
OK, date it is.

**Comment by angievanhegan** [comment]
@russ-manta @nadia847 I copied across some requirements from the work package on the RT board and put them at the top. I'm not sure they belong in this task though. Please can you check? Thank you.

**Comment by nadia847** [comment]
@angievanhegan @russ-manta Translation is it's own card so I took that bit out. It's more to the content type build than the search.

**Comment by dilwarhussain** [comment]
No longer having 'Quick search' and 'Advanced search' tabs. Quick search will always be displayed, there will be a button to expand and show 'Advanced search' options.

**Comment by Maria-alnap** [comment]
@ican-icant on the portals in the HELP homepage, we need to explain what they are and the fact that organisations can create their own if they wanted to. We should have something on the "about" menu on this, but we also think a bit of explanation needs to feature here somehow, maybe through a ? with a hover box or a text box underneath or something.

Open to your suggestions!

**Comment by angievanhegan** [comment]
@Maria-alnap Seeing as we're so close to the end of the week, I'm going to create this as a new issue to do later. I hope that's ok. It will get done! Thanks

**Comment by Maria-alnap** [comment]
@ican-icant: it occurred to me whilst presenting the site the other day that we need to better explain what portals are and the fact that organisations can create their own if they wanted to. We should have something on the "about" menu on this, but we also think a bit of explanation needs to feature in the portals on the HELP homepage somehow, maybe through a ? with a hover box or a text box underneath or something.

Open to your suggestions!

(I've added this same comment on Invision)

**Comment by ican-icant** [comment]
@Maria-alnap It's possible to add a short description below the portal title and link to 'portals'  under 'about' where there will be room for a full description. 

There was no description before, is this from recent feedback or because you want people to know they can set up a portal with you?

**Comment by Maria-alnap** [comment]
@ican-icant It's a bit of both. Just brainstorming a bit further now and we think we can just add this as the first thing under the about menu in each portal. So no need to do anything. Apologies for the false lead!

**Comment by angievanhegan** [comment]
@Maria-alnap Please can you confirm which internal tags need to be retained? Thanks.

**Comment by Maria-alnap** [comment]
@angievanhegan I've been looking at the backend and I think we would need to keep:
- ALNAP Steering Committee
- ECB (This is a portal, so I'm not sure we'll need to be internal tags any longer)
- Events and its subtags
- Publications and its subtags

**Comment by angievanhegan** [comment]
Thanks @Maria-alnap I'll move this to the requirements at the top of the issue.

**Comment by russ-manta** [comment]
@meldera can you look at these issues please in the morning: 
1. 'ALNAP resources to show at topof view , and then sorted by relevance'  
2. Number of resources at top of page do not update on submit but these are buttons to filter by All / ALNAP resources / Evaluative resources. Buttons disappear on submit via buttons or using keyword/other search. Submit a resource button stays.

**Comment by russ-manta** [comment]
Russ to discuss sorting options with Maria

**Comment by russ-manta** [comment]
@meldera looks like we may need to rebuild this using search api now.  We'll discuss more on Monday.

**Comment by angievanhegan** [comment]
FROM SLACK :

RUSS: for the HELP search it was previously mentioned that you might like to list ALNAP resources first followed by other resources.  Is this still the case? If so won't the other resources appear pages and pages back after the ALNAP resources?

MARIA: Yes, this is still the case. I don't think that we have that many resources under each tag so the other results shouldn't come that much further down. And if they do, people can always narrow down the search so I wouldn't worry about that (edited)

RUSS: ok, this will also affect the search when no filtering is performed though so without a filter applied all the ALNAP resources appear first too.

MARIA:  that's fine, thanks!

**Comment by russ-manta** [comment]
Additional done criteria are not finished - moving back to the sprint backlog

**Comment by angievanhegan** [comment]
@Maria-alnap  Can you  tell us how you'd mark a resource as 'ALNAP' in your current system? Thanks

**Comment by nadia847** [comment]
Search API's incompatibility with selective filters making the search not quite as slick as other search pages.

**Comment by russ-manta** [comment]
NB added new task to enable searching for items tagged similarly to the ones you get when searching for a keyword
